R/gibbs_draws.R

```diff
@@ -528,16 +528,21 @@ compute_t_min <- function(incid, si_distr, miss_at_most) {
 #' @param precompute a boolean (defaulting to TRUE) deciding whether to
 #'   precompute quantities or not. Using TRUE will make the algorithm faster
 #'
-#' @return a list with two elements.
+#' @return a list with three elements.
 #'   1) `epsilon` is a matrix containing the MCMC chain (thinned and after
 #'   burnin) for the relative transmissibility of the "new"
 #'   pathogen/strain/variant(s) compared to the reference
 #'   pathogen/strain/variant. Each row in the matrix is a "new"
 #'   pathogen/strain/variant and each column an iteration of the MCMC.
-#'   2) `R_out` is an array containing the MCMC chain (thinned and after
+#'   2) `R` is an array containing the MCMC chain (thinned and after
 #'   burnin) for the reproduction number for the reference
 #'   pathogen/strain/variant. The first dimension of the array is time,
 #'   the second location, and the third iteration of the MCMC.
+#'   3) `convergence` is a logical vector based on the results of a
```

Comment:
@j-wardle could you add something to say the convergence check is Gelman Rubin pls in the doc?